plugins/rhacm/src/components/ClusterStatusCard/ClusterStatusCard.tsx

```diff
@@ -12,26 +12,25 @@ import { useCluster } from '../ClusterContext';
 
 export const ClusterStatusCard = (): any => {
   const { data, loading, error } = useCluster();
-  let reason = data.status.reason;
+  let reason: string;
+  let down = false;
+  let available = false;
 
   if (error) {
-    data.status.reason = 'Unavailable';
+    reason = 'Unavailable';
```

Comment:
Why do you need to change the original error here? If you're returned `error` property, the reason should be already present - explaining what happened.